src/agent/src/main.rs

```diff
@@ -83,6 +85,24 @@ cfg_if! {
 
 const NAME: &str = "kata-agent";
 
+const OCICRYPT_CONFIG_PATH: &str = "/tmp/ocicrypt_config.json";
+const AA_PATH: &str = "/usr/local/bin/attestation-agent";
+const AA_UNIX_SOCKET_DIR: &str = "/run/confidential-containers/attestation-agent/";
+const UNIX_SOCKET_PREFIX: &str = "unix://";
+const AA_KEYPROVIDER_URI: &str =
+    concatcp!(UNIX_SOCKET_PREFIX, AA_UNIX_SOCKET_DIR, "keyprovider.sock");
+const AA_GETRESOURCE_URI: &str =
+    concatcp!(UNIX_SOCKET_PREFIX, AA_UNIX_SOCKET_DIR, "getresource.sock");
+const AA_ATTESTATION_SOCKET: &str = concatcp!(AA_UNIX_SOCKET_DIR, "attestation-agent.sock");
+const AA_ATTESTATION_URI: &str = concatcp!(UNIX_SOCKET_PREFIX, AA_ATTESTATION_SOCKET);
+
+cfg_if! {
```

Comment:
I'm a bit confused by the relationship between the AA and the CDH here. I am guessing that this is just a temporary measure before we remove the keyprovider stuff from the AA? 

In the longterm won't the AA and CDH always be enabled together or are there certain platforms where we would only want one? Do we want to keep using the `aa-kbc-params` check to determine if they should both be enabled?

Reply:
yes, maybe after refactor of attestation-agent, CDH can call AA as a library, but current we need talk with AA directly.
we use aa_kbc_params as KBS config for both CDH and AA, but CDH depends on AA.

Reply:
Ok. I guess this is fine for now.